src/spectrecoff-cli/commands/Dumpify.fs

```diff
@@ -0,0 +1,51 @@
+namespace SpectreCoff.Cli.Commands
+
+open Dumpify
+open Spectre.Console.Cli
+open SpectreCoff
+open SpectreCoff.Dumpify
+
+type DumpifySettings() =
+    inherit CommandSettings()
+
+type DumpifyExample() =
+    inherit Command<DumpifySettings>()
+    interface ICommandLimiter<DumpifySettings>
+
+    override _.Execute(_context, _settings) =
+        // use the dump function to print all kinds of objects
+        let list = [|1;2;3|]
```

Comment:
`[|1;2;3|] |> dump |> ignore` for consistency?

Reply:
Done